modules/lambda/main.tf

```diff
@@ -54,8 +54,8 @@ resource "aws_lambda_function" "lambda_vpc" {
   #reserved_concurrent_executions = "${var.reserved_concurrent_executions}"
 
   vpc_config {
-    subnet_ids         = ["${var.vpc_subnet_ids}"]
-    security_group_ids = ["${var.vpc_security_group_ids}"]
+    subnet_ids         = "${var.vpc_subnet_ids}"
```

Comment:
I didn't think this would work with 0.11, arrays still need to be wrapped there. Did you try a plan with these options set? AFAIK, none of the pandata usage uses it

Reply:
I did run a plan after seeing that option. Output was as expected; no errors.

I did some research to try to reconcile what you are saying with that I got experimentally, and I think the answer is here: https://github.com/hashicorp/terraform/issues/20220#issuecomment-466219345. In part:

"In older versions of Terraform these redundant brackets were required to signal to Terraform that a list was intended. This was made optional in a 0.10 release with the intent of it being phased out, but then this technique re-emerged later as a workaround for a bug in the 0.11 SDK with unknown values... This should work in 0.11, as long as this wasn't an example of working around the aforementioned old SDK bug, in which case it would probably cause the error "a list is required" when running in 0.11." 

Since I didn't get such an error running a plan, I'm guessing we were doing this out of an abundance of caution, and it wasn't actually needed here.

If we wanted  to be really careful, we could call this a breaking change and tag the release as such. 👍

Reply:
seems sane to me, happy with that as is